src/main/webapp/app/modeling-assessment-editor/modeling-assessment-editor.component.html

```diff
@@ -4,6 +4,7 @@ <h2 class="top-container">
         <fa-icon *ngIf="showBackButton" [icon]="'arrow-left'" (click)="goToExerciseDashboard()" class="back-button mr-2"></fa-icon>
         <div class="row-container" jhiTranslate="modelingAssessmentEditor.assessment">
             Assessment
+            <jhi-updating-result *ngIf="result?.rated && result.participation" [participation]="result.participation"></jhi-updating-result>
```

Comment:
Please remove this line again as the score is displayed in another way here.

Reply:
done